spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/support/NoOpTaskExplorer.java

```diff
@@ -22,6 +22,9 @@
 
```

Comment:
Do we still need this class if we have the `Map` implementation of the `TaskRepository`?